grails-app/controllers/org/pih/warehouse/SentryInterceptor.groovy

```diff
@@ -9,42 +9,60 @@
  **/
 package org.pih.warehouse
 
-
 import grails.util.Environment
-import io.sentry.SentryClient
-import io.sentry.event.UserBuilder
+import groovy.transform.CompileStatic
+import io.sentry.Sentry
+import io.sentry.protocol.User as SentryUser
```

Comment:
I've never understand what these blocks are. Do you have any ideas?

![image](https://user-images.githubusercontent.com/1162935/199814036-02ba319c-acd4-42ab-8600-875e30eb1b9e.png)

Reply:
@jmiranda my guess is that GitHub simply doesn't like `import ... as ...`. Those spaces are no different than the other spaces on the line, so it doesn't look like a "hey, you're using a weird encoding here"